platform-cloud/docs/compute-envs/seqera-compute.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: "Seqera Compute"
+description: "Instructions to set up Seqera Compute in Seqera Platform"
+date: "21 Apr 2025"
+tags: [seqera compute, compute environment]
+---
+
+Seqera Compute enables you to run pipelines and studio sessions in Seqera Cloud in a fully managed and optimized AWS environment. Seqera automatically provisions and manages all the underlying resources, including AWS accounts, credentials, roles, AWS Batch compute environments, and S3 storage buckets, requiring minimal user configuration. Using prepaid credits enables control and visibility of the compute spend in each of your organization's workspaces.
+
+### Manage Seqera Compute credits 
+
+Seqera Compute environments consume credits when running pipelines or studio sessions. Credits are consumed for CPU time, memory and storage usage, and network costs. 1 Seqera Compute credit is equivalent to $1 (USD), and resources are charged at the following rates:
+- CPU time: 1 CPU/Hr = 0.1 USD
+- Memory: 1 GiB/Hr = 0.025 USD 
+- Storage: 1 GB = 0.25 USD per month 
+
+:::note 
+Storage and network costs vary per region. Data ingress and egress across regions incure additional cost. 
```

Comment:
Can we note that Seqera Compute will charge ingress/egress at the stated AWS rate?